compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -52,9 +56,9 @@ static llvm::cl::opt<bool> clEnableVectorizationPasses(
 
 void appendVectorizationToPipeline(OpPassManager &pm) {
```

Comment:
Nit: upstream I tried to capture the nesting in the name.there is no other way to know what the nesting is though unfortunately.

Reply:
makes sense, changed the passManager's name to funcPassManager so the nesting is clear.